Gemfile

```diff
@@ -0,0 +1,4 @@
+# A sample Gemfile
+source "https://rubygems.org"
+
+gem "chunky_png"
```

Comment:
This is great, thanks! Do you think we could specify the version? and also add `byebug` in there? I have the `require` commented out so it's easy for someone to quickly start debugging. I can update the `README.md` afterwards.